src/strava_offline/reports.py

```diff
@@ -1,11 +1,31 @@
+from io import StringIO
 import sqlite3
+import csv
+from typing import Optional
 
 from tabulate import tabulate
 
 
-def tabulate_execute(db: sqlite3.Connection, sql: str, *params) -> str:
+def tabulate_execute(db: sqlite3.Connection, sql: str, *params, format='plain') -> str:
+    # db.set_trace_callback(print)
+    # sqlite3.enable_callback_tracebacks(True)
     table = (dict(row) for row in db.execute(sql, params))
-    return tabulate(table, headers='keys')
+    if format == 'csv':
```

Comment:
Can we turn this into a separate commit that adds configurable output formats to all reports, pls? You can steal some code from here: https://github.com/liskin/strava-gear/blob/0b6195ca6e08a8790998d2f0e756aba682a6d93c/src/strava_gear/cli.py#L42-L47
Also the strava-gear code passes the output stream to the report function instead of using `StringIO`, and that's probably better as well. I'll be happy to do this refactoring myself later but if you want to help I wouldn't mind :-)